blog/forge-v6-release.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Electron Forge v6 Stable Release
+date: 2022-10-26T00:00:00.000Z
+authors:
+  - name: georgexu99
+    url: 'https://github.com/georgexu99'
+    image_url: 'https://github.com/georgexu99.png?size=96'
+  - name: vertedinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: forge-v6-release-blog
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Electron Forge Team is excited to announce that Forge v6 is now available! This stable release contains many significant new features and improvements. Highlights of this release include the addition of our Webpack plugin as well as the introduction of the Forge Core and CLI APIs.
+
+## What Does Switching to Stable Mean?
+
+During our time in a beta release state, our team has been busy refactoring Forge’s code, improving its architecture, creating new features as well as ironing out bugs and issues.
+
+Now that we are done making changes, we’re ready to release a stable version of Forge!
```

Comment:
> Now that we are done making changes

Is this the answer to "What Does Switching to Stable Mean"? Doesn't seem right to me.

Reply:
"now that we are done making _major breaking_ changes"?